app/flows/check_travel_during_coronavirus_flow/outcomes/_country.erb

```diff
@@ -1,23 +1,37 @@
+<% heading = capture do %>
+  Read entry requirements for <%= country.title %>
+
+  <% if country.on_red_list? %>
+    <strong class='govuk-tag govuk-tag--red'>RED LIST</strong>
+  <% end %>
+<% end %>
+
 <%= render "govuk_publishing_components/components/heading", {
-  text: country.title,
+  text: heading,
   heading_level: 3,
   font_size: "m",
   margin_bottom: 4,
 } %>
 
+<% if country.on_red_list? %>
+  <p class="govuk-body"><%= country.title %> is on the travel red list.</p>
+<% end %>
+
 <% if country.next_covid_status.present? %>
   <%= render partial: "country/status_change", locals: { country: country } %>
 <% end %>
 
 <% if calculator.countries_with_content_headers_converted.include?(country.slug) %>
-  <p class="govuk-body">You should read the following sections of the <%= country.title %> entry requirements guidance on:</p>
+  <p class="govuk-body">You should read the <%= country.title %> entry requirements guidance. Based on your answers, relevant sections are:</p>
 
   <% if calculator.transit_countries.include?(country.slug) %>
     <ul class="govuk-list govuk-list--bullet">
       <li><a href="https://www.gov.uk/foreign-travel-advice/<%= country.slug %>/entry-requirements#if-youre-transiting-through-<%= country.slug %>" class="govuk-link">travelling through <%= country.title %></a></li>
     </ul>
   <% else %>
     <ul class="govuk-list govuk-list--bullet">
+      <li><a href="https://www.gov.uk/foreign-travel-advice/<%= country.slug %>/entry-requirements#all-travellers" class="govuk-link" rel="noreferrer noopener" target="_blank">all travellers</a> (opens in a new tab)</li>
```

Comment:
I noticed that '(opens in a new tab)' is on the wrong side of the `</a>` so its rendering as plain text rather than part of the link text. It should be part of the link text according to the [design system](https://design-system.service.gov.uk/styles/typography/#links) (the 'Opening links in a new tab' heading).

Reply:
and thanks for explaining in the commit about the other new tab stuff being in Jon's PR. Hopefully the rebase won't be too painful!

Reply:
Thank you! It looks like the wording should be “opens in new tab” rather than “opens in a new tab” too, so I'll update both.

Reply:
👍